client/pages/CalendarScreen/EventManager/DescriptionInput.tsx

```diff
@@ -0,0 +1,91 @@
+import { eventNames } from 'process';
+import React from 'react';
+import { Button, Modal, StyleSheet, Text, Pressable, View, TextInput } from 'react-native';
+import DateTimePicker from '@react-native-community/datetimepicker';
+
+const ModalPopup = ({ visible, save, cancel, children }) => {
+    const [showModal, setShowModal] = React.useState(visible);
+
+    return (
+        <Modal transparent visible={true}>
+            <View style={styles.modalBackground}>
+                <View style={[styles.modalContainer]}>
+                    <View style={styles.header}>
+                        <Button title="Cancel" onPress={cancel} />
+                        <Button title="Save" onPress={save} />
+                    </View>
+                    {children}
+                </View>
+            </View>
+        </Modal>
+    );
+};
+
+const App = () => {
```

Comment:
Just put `export default () => {`, you dont need to store it in a variable and export it below @faris-kazi